src/ShopBundle/objects/db/TShopPaymentHandler/TShopPaymentHandlerPayPal_PayViaLink.class.php

```diff
@@ -144,7 +144,16 @@ public function ProcessIPNRequest($oOrder, $aURLParameter)
         $oTools = ServiceLocator::get('chameleon_system_core.tools');
         $sResponse = $oTools::sendToHost($sDomain, 'POST', $sPath, $sData, false, 'application/x-www-form-urlencoded', true);
         if (0 != strcmp($sResponse, 'VERIFIED')) {
-            $logger->error("PayPal IPN: unable to send notify-validate response. Domain:{$sDomain}\nPath:{$sPath}\nParameter: ".print_r($aURLParameter, true)."\ndata: {$sData}\nRESPONSE: ".$sResponse);
+            $logger->error(
+                'PayPal IPN: unable to send notify-validate response.',
+                [
+                    'domain' => $sDomain,
+                    'path' => $sPath,
+                    'url_parameters' => $aURLParameter,
+                    'data' => $sData,
+                    'response' => $sResponse,
```

Comment:
please reference the order (id and ordernumber) - both are very helpful when searching for a problem with an ordern

Reply:
ok